### PR TITLE
Update actions for main branch 

### PR DESCRIPTION
**Pull request by sung-su**
### Description of Change ###

- Update Action according to the change of the default branch name from `master` to `main`.
  - This should be merged after the default branch name is changed to `main`.
- Removed `Download Base API Spec` step on build-PR actions.

### CODE REVIEW

**Comment by JoonghyunCho** [comment]
Any reason you left `Download Base API Spec` as commented ?

**Comment by sung-su** [comment]
Update done.

File: .github/workflows/build-pull-request.yml [hunk 1]
```diff
@@ -78,37 +78,25 @@ jobs:
...
-      uses: TizenAPI/tizenfx-build-actions/s3-download-public@master
-      with:
-        region: ap-northeast-2
-        bucket: tizenfx-api-specs
-        key: ${{ github.event.pull_request.base.sha }}.json
-        path: Artifacts/base-api.json
-
-    - uses: actions/checkout@v3
-      if: ${{ steps.base-spec.outputs.code != 200 }}
+    - name: PR event trigger
```
Comment (JoonghyunCho) [inline .github/workflows/build-pull-request.yml:81]: Maybe it is better to name it something like "Checkout to base spec"? "Checkout to base  API spec"?

Comment (sung-su) [reply .github/workflows/build-pull-request.yml:81]: OK `Checkout to base spec` is better

Comment (JoonghyunCho) [reply .github/workflows/build-pull-request.yml:81]: Thanks!

**Pull request merged**